package.json

```diff
@@ -103,6 +106,7 @@
     "pretest": "npm run lint-files",
     "lint-files": "npm run lint && npm run schema-lint",
```

Comment:
call the typescript linting here, to add it to the CI

Reply:
Will do once number of errors is reduced. The lint bot comments will pollute this thread if I do. See https://github.com/webpack/webpack/pull/6869